framework/db/Connection.php

```diff
@@ -132,7 +132,7 @@
  * @author Qiang Xue <qiang.xue@gmail.com>
  * @since 2.0
  */
-class Connection extends Component
+class Connection extends Component implements ConnectionInterface
```

Comment:
It doesn't belong to this pull request.

Reply:
sure, I've branch it from the Connection one. I'll fix this.

Reply:
@samdark rebased UrlManager into master parent, can you review if everything is ok? thanks!